initializr-service/src/main/resources/application.yml

```diff
@@ -951,6 +958,67 @@ initializr:
           description: Hystrix circuit breaker on Pivotal Cloud Foundry
           groupId: io.pivotal.spring.cloud
           artifactId: spring-cloud-services-starter-circuit-breaker
+    - name: Azure
+      bom: azure
+      versionRange: 1.5.4.RELEASE
+      content:
+        - name: Azure Active Directory
+          id: active-directory
```

Comment:
should be `azure-ad-integration` (or whatever the prefix of the starter will be)